schematic_db/schema/schema.py

```diff
@@ -19,9 +19,18 @@
     get_project_manifests,
     get_manifest,
     is_node_required,
+    get_manifest_datatypes,
     ManifestSynapseConfig,
 )
 
+DATATYPES = {
+    "string": DBDatatype.TEXT,
+    "object": DBDatatype.TEXT,
```

Comment:
Just a heads up that pandas will also return type object when there are multiple non-string datatypes in a column, ie ints and floats

Reply:
Yep! Text types will handle all of those.

Reply:
Cool!